app/microserviceshandler.py

```diff
@@ -211,11 +215,17 @@ async def get(self, request):
 
       views, current_view_index = get_views(user_role, microservice_name)
       header_html = get_html(user_role, views)
+      current_view = views[current_view_index]
+      if (current_view.database_name == "search") and (page_number == 1):
+        current_view.hide_table = True
+      elif current_view.database_name == "search" and page_number > 1:
+        current_view.hide_table = False
+        current_view.hide_search_criteria = True
```

Comment:
The above could be simplified.
```python
current_view = views[current_view_index]
if current_view.database_name == 'search':
  current_view.hide_table = page_number == 1
  current_view.hide_search_criteria = page_number > 1
```

Reply:
Have swapped out the logic for this better statement 👌